keras_nlp/src/models/resnet/resnet_backbone.py

```diff
@@ -580,7 +678,12 @@ def apply_stack(
         Output tensor for the stacked blocks.
     """
     if name is None:
-        version = "v1" if not use_pre_activation else "v2"
+        if use_vd_pooling:
```

Comment:
Similar to above I think we should remove this. The preset name is how we can track different in vd vs v2 vs v2. This will be hard to maintain.